ramalama/model_inspect/gguf_parser.py

```diff
@@ -42,6 +42,16 @@ class GGML_TYPE(IntEnum):
     GGML_TYPE_I64 = (27,)
     GGML_TYPE_F64 = (28,)
     GGML_TYPE_IQ1_M = (29,)
+    GGML_TYPE_BF16 = (30,)
+    # GGML_TYPE_Q4_0_4_4 = 31, support has been removed from gguf files
+    # GGML_TYPE_Q4_0_4_8 = 32,
+    # GGML_TYPE_Q4_0_8_8 = 33,
+    GGML_TYPE_TQ1_0 = (34,)
+    GGML_TYPE_TQ2_0 = (35,)
+    # GGML_TYPE_IQ4_NL_4_4 = 36,
+    # GGML_TYPE_IQ4_NL_4_8 = 37,
+    # GGML_TYPE_IQ4_NL_8_8 = 38,
+    GGML_TYPE_MXFP4 = (39,)  # MXFP4 (1 block)
```

Comment:
It seems the doc here is not yet updated since I can't find these types there:
https://github.com/ggml-org/ggml/blob/master/docs/gguf.md#file-structure

`GGML_TYPE_BF16`, `GGML_TYPE_TQ1_0` and `GGML_TYPE_TQ2_0` I can find here:
https://github.com/ggml-org/ggml/blob/master/include/ggml.h#L358-L399

Where is `GGML_TYPE_MXFP4` coming from? Position 39 should `GGML_TYPE_COUNT`.

Reply:
Ah ok, thanks! Cutting edge changes.